src/modules/new-request-form/translations/en-us.yml

```diff
@@ -0,0 +1,134 @@
+      value: "Upload error"
+  - translation:
+      key: "new-request-form.attachments.upload-error-description"
+      title: "Description of the notification shown when the upload of an attachment failed. {{fileName}} will be replaced by file name, e.g., [image.png]. Alternative English if you encounter declension issues: [There was an error uploading the file {{fileName}}. Try again or upload another file.] "
+      screenshot: "https://drive.google.com/file/d/19pYyTsdX_PzxZpMmN3-1sQn7gG_3bxl6/view?usp=drive_link"
+      value: "There was an error uploading {{fileName}}. Try again or upload another file."
+  - translation:
+      key: "new-request-form.attachments.drop-files-label"
+      title: "Label shown when a file is dragged over the attachment field"
+      screenshot: "https://drive.google.com/file/d/1T_2Eahrfs8iITZWvgAR6wxrxEdsy2D2G/view?usp=drive_link"
+      value: "Drop files here"
+  - translation:
+      key: "new-request-form.attachments.choose-file-label"
+      title: "Label for the attachments field"
+      screenshot: "https://drive.google.com/file/d/1T_2Eahrfs8iITZWvgAR6wxrxEdsy2D2G/view?usp=drive_link"
+      value: "Choose a file or drag and drop here"
+  - translation:
+      key: "new-request-form.attachments.stop-upload"
+      title: "Tooltip for the button for stopping the upload of an attachment"
+      screenshot: "https://drive.google.com/file/d/1Y6yeqp3fbfJbuq7tT8VutsLa80GZju-V/view?usp=drive_link"
+      value: "Stop upload"
+  - translation:
+      key: "new-request-form.attachments.remove-file"
+      title: "Tooltip for the button for removing an attachment"
+      screenshot: "https://drive.google.com/file/d/1JFVE2twzUkg-U6gGAs2EHqD-3jPvtJDA/view?usp=drive_link"
+      value: "Remove file"
+  - translation:
+      key: "new-request-form.attachments.uploading"
+      title: "[A11Y] Hidden label for screen readers for the upload attachment progress bar. {{fileName}} will be replaced by file name, e.g., [image.png]. Alternative English if you encounter declension issues: [Uploading the file {{fileName}}]"
+      screenshot: "https://drive.google.com/file/d/1Y6yeqp3fbfJbuq7tT8VutsLa80GZju-V/view?usp=drive_link"
+      value: "Uploading {{fileName}}"
+  - translation:
+      key: "new-request-form.submit"
+      title: "Submit button for the New Request Form"
+      screenshot: "https://drive.google.com/file/d/1VuVYpxGA-uAGbvscl2WXTJ24yK5QydhZ/view?usp=drive_link"
+      value: "Submit"
+  - translation:
+      key: "new-request-form.cc-field.email-removed"
+      title: "[A11Y] Announcement read by screen readers when an e-mail is removed from the CC field. {{email}} will be replaced by an email address, e.g., [test@example.com]. Alternative English if you encounter declension issues: [The email address {{email}} has been removed]"
+      screenshot: "https://drive.google.com/file/d/19iWawEvm6PJw1D2M7I1b32ntMDb7GqNw/view?usp=drive_link"
+      value: "{{email}} has been removed"
```

Comment:
@Fredx87 We don't have the plural version of that one anymore ^^

Reply:
There wasn't a plural version of this one, a user can add multiple emails at once (for example pasting a list of emails separated by commas), but they cannot remove multiple emails at once, just one by one